qcelemental/molparse/from_arrays.py

```diff
@@ -353,37 +353,34 @@ def from_arrays(*,
             update_with_error(molinit, processed)
             nat = molinit['geom'].shape[0] // 3
 
-        processed = validate_and_fill_nuclei(
-            nat,
-            elea=elea,
-            elez=elez,
-            elem=elem,
-            mass=mass,
-            real=real,
-            elbl=elbl,
-            speclabel=speclabel,
-            nonphysical=nonphysical,
-            mtol=mtol,
-            verbose=verbose)
+        processed = validate_and_fill_nuclei(nat,
+                                             elea=elea,
+                                             elez=elez,
+                                             elem=elem,
+                                             mass=mass,
+                                             real=real,
+                                             elbl=elbl,
+                                             speclabel=speclabel,
+                                             nonphysical=nonphysical,
+                                             mtol=mtol,
+                                             verbose=verbose)
         update_with_error(molinit, processed)
 
-        processed = validate_and_fill_fragments(
-            nat,
-            fragment_separators=fragment_separators,
-            fragment_charges=fragment_charges,
-            fragment_multiplicities=fragment_multiplicities)
+        processed = validate_and_fill_fragments(nat,
+                                                fragment_separators=fragment_separators,
+                                                fragment_charges=fragment_charges,
+                                                fragment_multiplicities=fragment_multiplicities)
         update_with_error(molinit, processed)
 
         Z_available = molinit['elez'] * molinit['real'] * 1.
-        processed = validate_and_fill_chgmult(
-            zeff=Z_available,
-            fragment_separators=molinit['fragment_separators'],
-            molecular_charge=molecular_charge,
-            fragment_charges=molinit['fragment_charges'],
-            molecular_multiplicity=molecular_multiplicity,
-            fragment_multiplicities=molinit['fragment_multiplicities'],
-            zero_ghost_fragments=zero_ghost_fragments,
-            verbose=verbose)
+        processed = validate_and_fill_chgmult(zeff=Z_available,
+                                              fragment_separators=molinit['fragment_separators'],
+                                              molecular_charge=molecular_charge,
+                                              fragment_charges=molinit['fragment_charges'],
+                                              molecular_multiplicity=molecular_multiplicity,
+                                              fragment_multiplicities=molinit['fragment_multiplicities'],
+                                              zero_ghost_fragments=zero_ghost_fragments,
+                                              verbose=verbose)
```

Comment:
@Lnaden check just once that your yapf is up-to-date. Daniel and I had dueling yapfs for a while.

Reply:
I just installed/updated from c-f before doing this, so I should be good.